app/views/commontator/comments/_body.html.erb

```diff
@@ -8,10 +8,11 @@
 <!--- Show timestamps in comments that have a linked annotation --->
 <% annotation = comment.annotation %>
 <% unless annotation.nil? %>
-  <% medium = annotation.medium %>
+  <% medium_id = annotation.medium.id %>
   <% timestamp = annotation.timestamp %>
   <br>
   Timestamp:
-  <a href= '<%= Rails.application.routes.url_helpers.play_medium_path(medium) %>?time=<%= timestamp.total_seconds %>'>
-  <%= timestamp.hms_colon_string %> </a>
+  <a href="<%= video_link_timed(medium_id, timestamp) %>">
```

Comment:
Why not use the Rails helper for links?
`<%= link_to video_link_timed(medium, timestamp) do ... %>`
reads nicer (at least to me).

Reply:
ups, forgot to replace that occurrence thanks

Reply:
Ah, now I see what you meant, use `link_to` instead of the custom anchor link tag `<a>`. That's just my personal preference which probably stems from me not having available proper syntax highlighting for ERB syntax yet, while for `<a>` I get the well-established HTML syntax highlighting. But still changed it to `link_to` and hopefully we can find a good HTML ERB linter in the future.